tools/userchroot.nix

```diff
@@ -0,0 +1,60 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
Could you add a disclaimer of sorts here so that people know to reach out to you for support? I don't recall anybody else working with this project using NixOS. Something like:

-- This script is provided by @grwlf and is supported on best effort basis. TensorFlow development team does
-- not regularly test this script and can't answer any questions about it.